spec/requests/internal/chat_channel_spec.rb

```diff
@@ -21,4 +21,14 @@
       expect(ChatChannel.last.pending_users).to include(user)
     end
   end
+
+  describe "DELETE /internal/chat_channels/:id/remove_user" do
+    it "removes the user from the chat channel" do
+      delete "/internal/chat_channels/#{user.id}/remove_user",
+             params: { chat_channel: { usernames_string: user.username.to_s } }
+    end
+
+    expect(user.chat_channel_memberships.count).to eq 0
+    expect(ChatChannel.last.pending_users).to include(user)
```

Comment:
I'm a bit confused here; why would the user be a pending user after being removed?

Reply:
This was my first time doing the test, I was confused of doing that. I will try to fix that.

Reply:
No worries! Feel free to ask questions :)

Reply:
@Zhao-Andy  I have updated the test and pass successfully. May you check for me.